safe_extras/parameters/numeric_parameter.py

```diff
@@ -9,9 +9,6 @@
 class NumericParameter(GenericParameter):
     """A subclass of generic parameter that accepts float or integer only.
 
-    Known issue : You should set the minimum and the maximum value explicitly.
-    See https://github.com/inasafe/inasafe/issues/2468#issuecomment-171124507
-
```

Comment:
yes, I made some changes last year in inasafe/inasafe and i wasn't aware about inasafe/parameter repo.